src/main/kotlin/no/nav/syfo/infrastructure/journalforing/JournalforingService.kt

```diff
@@ -5,10 +5,14 @@ import no.nav.syfo.domain.*
 import no.nav.syfo.infrastructure.clients.dokarkiv.DokarkivClient
 import no.nav.syfo.infrastructure.clients.dokarkiv.dto.*
 import no.nav.syfo.infrastructure.clients.pdl.PdlClient
+import org.slf4j.LoggerFactory
+
+const val DEFAULT_FAILED_JP_ID = 0
 
 class JournalforingService(
     private val dokarkivClient: DokarkivClient,
     private val pdlClient: PdlClient,
+    private val journalforingRetryEnabeled: Boolean,
```

Comment:
Liten typo + forslag om `is…` foran for å tydeliggjøre at det er `Boolean` 😊
```suggestion
    private val isJournalforingRetryEnabled: Boolean,
```